cognite/power/data_classes.py

```diff
@@ -866,8 +879,8 @@ def load_duration_curve(
         index_granularity=0.1,
     ) -> "pd.DataFrame":
         """See ACLineSegment#load_duration_curve"""
-        if self.type not in ["ACLineSegment", "PowerTransformerEnd", "SynchronousMachine"]:
-            raise WrongPowerTypeError(f"Can't get connected current limits dataframe for a list of {self.type}")
+        if self.type not in ["ACLineSegment", "PowerTransformerEnd", "SynchronousMachine"]:  # , "PowerTransferCorridor"
```

Comment:
does this comment mean anything?

Reply:
it means i added it and then removed it and we probably need to add it again :)